addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -296,6 +296,32 @@ def _update_invoice_from_binary(self, filename, content, extension, invoice):
                 return self._import_fattura_pa(decoded_content, invoice)
         return super()._update_invoice_from_binary(filename, content, extension, invoice)
 
+    def get_partner_invoice(self, tree, company):
```

Comment:
```suggestion
    def _l10n_it_get_partner_invoice(self, tree, company):
```
We should make the method private and give it a more specific name

Reply:
is it ready now?

Reply:
> We should make the method private and give it a more specific name

@jco-odoo this has been done, can you update your review? Thanks!